react/components/ProductImages/components/Carousel/index.js

```diff
@@ -17,7 +17,9 @@ import './global.css'
 
 import Gallery from '../Gallery'
 
-import Swiper, { Pagination, Navigation } from 'react-id-swiper/lib/ReactIdSwiper.full'
+const Swiper = window.navigator ? require('react-id-swiper/lib/ReactIdSwiper.full').default : null
```

Comment:
talvez adiciona um comentariozinho ai em cima explicando o motivo disso, pra evitar mudarem no futuro

Reply:
```suggestion
/** Swiper and its modules are imported using require to avoid breaking SSR */
const Swiper = window.navigator ? require('react-id-swiper/lib/ReactIdSwiper.full').default : null
```